src/commands/heroku.rs

```diff
@@ -24,15 +26,162 @@ pub fn get_app(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandResul
+        },
+    )?;
+
+    let app_formations_resp =
+        heroku_client(ctx).request(&formations::FormationList { app_id: app_name });
+
+    msg.reply(
+        ctx,
+        match app_formations_resp {
+            Ok(formations) => app_formations_response(formations),
+            Err(e) => {
+                format!(
+                    "An error occured when fetching your Heroku app formation info:\n{}",
+                    e
+                )
+            }
+        },
+    )?;
+
+    Ok(())
+}
+
+// Config variables that can be updated through Discord
+// Set only as "FOO" until we fill this in with the real config vars 
+// for our Heroku account that we want to allow to be updated
+const AUTHORIZED_CONFIG_VARS: &[&str] = &["FOO"];
+
+// Get app by name or id
+#[command]
+#[num_args(3)]
+pub fn update_app_config(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandResult {
+    let app_name = args
+        .single::<String>()
+        .expect("You must include an app name");
+
+    let config_var_key = args
+        .single::<String>()
+        .expect("You must include a config variable key");
+
+    let config_var_key_reference: &str = &config_var_key;
+
+    let config_var_value = args
+        .single::<String>()
+        .expect("You must include a config variable value");
+
+    if AUTHORIZED_CONFIG_VARS.contains(&config_var_key_reference) {
+        let mut config_var = HashMap::new();
+        config_var.insert(config_var_key, config_var_value);
+
+        let response = heroku_client(ctx).request(&config_vars::AppConfigVarUpdate {
+            app_id: &app_name.clone(),
+            params: config_var.clone(),
+        });
+
+
+        msg.reply(
+            ctx,
+            match response {
+                Ok(_response) => format!("Config Var has been updated {:?}", config_var),
+                Err(e) => {
+                    format!(
+                        "An error occured when trying to restart your Heroku app:\n{}",
+                        e
+                    )
+                }
+            }
+        )?;
+    } else {
+        msg.reply(
+            ctx.clone(),
+            format!(
+                "Config var {} is not authorized to be updated from Discord",
+                &config_var_key
+            )
+        )?;
+    }
+
+    Ok(())
+}
+
+#[command]
+#[num_args(4)]
+pub fn scale_app(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandResult {
+    let app_name = args
+        .single::<String>()
+        .expect("You must include an app name");
+
+    let formation_name = args
+        .single::<String>()
+        .expect("You must include a formation name");
+
+    let quantity = args.single::<i32>().expect("You must include a quantity");
+
+    let size = args.single::<String>().expect("You must include a size");
+
+    let response = heroku_client(ctx).request(&formations::FormationUpdate {
+        app_id: app_name.clone(),
+        formation_id: formation_name.clone(),
```

Comment:
I feel like this one is necessary.

If I change it to:

```rust
    let response = heroku_client(ctx).request(&formations::FormationUpdate {
        app_id: &app_name,
        formation_id: formation_name.clone(),
        params: formations::FormationUpdateParams {
            quantity: Some(quantity),
            size: Some(size),
        },
    });
```
I receive an error because the Heroku client request method expects a std::string::String, rather than a &std::string::String.

If I change it to:

```rust
    let response = heroku_client(ctx).request(&formations::FormationUpdate {
        app_id: app_name,
        formation_id: formation_name.clone(),
        params: formations::FormationUpdateParams {
            quantity: Some(quantity),
            size: Some(size),
        },
    });
```

I get an error from further down in the code:

```bash
error[E0382]: use of moved value: `app_name`
   --> src/commands/heroku.rs:150:57
    |
126 |     let app_name = args
    |         -------- move occurs because `app_name` has type `std::string::String`, which does not implement the `Copy` trait
...
139 |         app_id: app_name,
    |                 -------- value moved here
...
150 |             Ok(formation) => formation_updated_response(app_name, formation),
    |                                                         ^^^^^^^^ value used here after move

error: aborting due to previous error

For more information about this error, try `rustc --explain E0382`.
error: could not compile `crates-io-ops-bot`.
```